packages/twenty-ui/src/theme/constants/SecondaryColors.ts

```diff
@@ -46,6 +47,7 @@ export const SECONDARY_COLORS = {
   blue30: '#a3c0f8',
   blue20: '#d1dffb',
   blue10: '#e8effd',
+  blueIcon: '#1961ed',
```

Comment:
Unsure about naming

Reply:
We should use the colors from: 

https://github.com/twentyhq/twenty/blob/1490c5bd4e5737f7568ae5aa5001169abbccc15c/packages/twenty-ui/src/theme/constants/MainColors.ts